src/app/[locale]/settings/_components/mnestix-connections/MnestixConnectionsCard.tsx

```diff
@@ -0,0 +1,134 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import { messages } from 'lib/i18n/localization';
+import { Box } from '@mui/material';
+import { useState } from 'react';
+import { useNotificationSpawner } from 'lib/hooks/UseNotificationSpawner';
+import {
+    getConnectionDataAction,
+    upsertConnectionDataAction,
+} from 'app/[locale]/settings/_components/mnestix-connections/MnestixConnectionServerActions';
+import { useForm } from 'react-hook-form';
+import { useEnv } from 'app/env/provider';
+import { SettingsCardHeader } from 'app/[locale]/settings/_components/SettingsCardHeader';
+import { MnestixConnectionsForm } from 'app/[locale]/settings/_components/mnestix-connections/MnestixConnectionForm';
+
+export type ConnectionFormData = {
+    aasRepository: {
+        id: string;
+        url: string;
+        type: string;
+    }[];
+    submodelRepository: {
+        id: string;
+        url: string;
+        type: string;
+    }[];
+};
+
+type DataSource = {
+    name: string;
+    url: string | undefined;
+};
+
+export function MnestixConnectionsCard() {
+    const notificationSpawner = useNotificationSpawner();
+    const [isEditMode, setIsEditMode] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+    const intl = useIntl();
+    const env = useEnv();
+
+    const dataSources: DataSource[] = [
+        { name: 'aasRepository', url: env.AAS_REPO_API_URL },
+        { name: 'submodelRepository', url: env.AAS_REPO_API_URL }, //TODO: Update env to SUBMODEL_REPO_API_URL after merge with "dev" branch
```

Comment:
So this is done after the PR is completed?

Reply:
I'll change it here because the dev branch was already merged into this feature branch